package.json

```diff
@@ -26,6 +26,7 @@
   },
   "scripts": {
     "build": "gulp build",
+    "full-test": "nyc mocha -r ts-node/register tests/**/*.spec.ts tests/**/**/*.spec.ts tests/**/**/**/*.spec.ts tests/**/*.e2e.ts tests/**/**/*.e2e.ts tests/**/**/**/*.e2e.ts",
     "start": "node ./bin/procbots -b ${PROCBOT_BOTS_TO_LOAD}",
     "test": "nyc mocha -r ts-node/register tests/**/*.spec.ts tests/**/**/*.spec.ts tests/**/**/**/*.spec.ts"
```

Comment:
Same as above goes for here, as well. You should only need:
```json
"test": "nyc mocha -r ts-node/register tests/**/*.spec.ts"
```

Reply:
@hedss, see above

Reply:
@sqweelygig See above above. :)